module/VuFind/src/VuFind/Search/NewItemsHelper.php

```diff
@@ -21,64 +21,78 @@
  * <https://www.gnu.org/licenses/>.
  *
  * @category VuFind
- * @package  Controller_Plugins
+ * @package  Search
  * @author   Demian Katz <demian.katz@villanova.edu>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
 
-namespace VuFind\Controller\Plugin;
+namespace VuFind\Search;
 
-use Laminas\Mvc\Controller\Plugin\AbstractPlugin;
 use Laminas\Mvc\Plugin\FlashMessenger\FlashMessenger;
 use VuFind\Config\Config;
+use VuFind\ILS\Connection;
 
 use function array_slice;
 use function count;
 use function intval;
 use function is_string;
 
 /**
- * Action helper to perform new items-related actions
+ * Helper to perform new items-related actions
  *
  * @category VuFind
- * @package  Controller_Plugins
+ * @package  Search
  * @author   Demian Katz <demian.katz@villanova.edu>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
-class NewItems extends AbstractPlugin
+class NewItemsHelper
 {
     /**
      * Configuration
      *
      * @var Config
      */
-    protected $config;
+    protected Config $config;
```

Comment:
We're working on phasing out Config objects and just using arrays instead. Maybe refactoring this class is an opportunity to remove Config objects from one more place. It shouldn't be too difficult to use `getConfigArray` instead of `getConfigObject` in the factory for the helper, and then switch from object syntax to array syntax in all of the references to $this->config.

If you think that's too much change all at once, we could handle it in a follow-up PR after this one is merged... or you can do it now if you prefer. I'll leave it up to you!